CHANGELOG.asciidoc

```diff
@@ -224,6 +224,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Docker and Kubernetes modules are now GA, instead of Beta. {pull}6105[6105]
 - Add pct calculated fields for Pod and container CPU and memory usages. {pull}6158[6158]
 - Add statefulset support to Kubernetes module. {pull}6236[6236]
+- Refactor prometheus endpoint parsing to look similar to upstream prometheus {pull}6332[6332]
```

Comment:
When I initially read this I thought the parsing of `prometheus input -> elasticsearch output` changed. But it seems mainly the error handling changed / was improved?

Reply:
it was always meant to change only the way parsing happens.